openid-4-verifiable-presentations-1_0.md

```diff
@@ -251,6 +251,10 @@ Depending on the Client Identifier Scheme, the Verifier can communicate a JSON o
 
 This specification enables the Verifier to send both Presentation Definition JSON object and Client Metadata JSON object by value or by reference.
 
+Additional request parameters, other than those defined in this section, MAY be defined and used, as described in [@!RFC6749].
+The Wallet MUST ignore any unrecognized parameters, other than the `transaction_data` parameter.
+The wallets that do not support the `transaction_data` parameter MUST reject requests that contain it.
```

Comment:
why is this in this paragraph and not in the new parameters section?

Reply:
because this is about extensibility. the previous sentence says ignore if you don't recognize a parameter, and the next sentence says, but for one specific `transaction_data` parameter, do not ignore (ie the rule in the previous sentence does not apply)

Reply:
i can clarify this